decidim/decidim
# Replace xls with xlsx

#### :tophat: What? Why?
This pull request updates Decidim to newer excel format and because of that, we have to replace spreadsheet gem with rubyXL gem.

#### :pushpin: Related Issues
https://meta.decidim.org/processes/roadmap/f/122/proposals/14921

#### Testing
1. Login as admin
2. Try to export something (in the excel format) from admin panel (e.g results from a accountability component)
3. Go to http://localhost:3000/letter_opener and download export
4. Import proposals (Example file can be found at decidim-dev/lib/decidim/dev/assets/import_proposals.xlsx)
5. Go to see imported proposals

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.adoc) to this repository.


:hearts: Thank you!
